package-lock.json

```diff
@@ -8331,8 +8331,7 @@
         },
```

Comment:
This should not be commited if there is no change to `package.json`

Reply:
There are some changes though they are not getting rendered here.

Reply:
I wrote `package.json` not `package-lock.json`. `package-lock.json` should not be updated if there is nothing changed to the packages. You can edit the commit integrating the change, or you can revert it to what it was and commit it. But that's a detail, if the rest is good, I can help you doing that before merging.